exercise/express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable indent */
+/* eslint-disable eol-last */
+/* eslint-disable semi */
+/* eslint-disable no-console */
+/* eslint-disable quotes */
+/* eslint-disable no-unused-vars */
```

Comment:
In general, please avoid turning off ESLint. The code still works most of the time if you see the red squiggles, you just need to fix the problems before committing.